qutebrowser/config/configdata.yml

```diff
@@ -2512,6 +2512,22 @@ colors.webpage.prefers_color_scheme_dark:
 
 # emacs: '
 
+## unkown_url
+
+unknown_url.scheme.policy:
```

Comment:
I don't think it makes sense to open a new "group" of settings with only this setting in it. What about `content.unknown_url_scheme_policy`?